src/headline.less

```diff
@@ -0,0 +1,51 @@
+.h1 {
+  font-size: 7rem;
+  font-weight: @font-medium;
```

Comment:
things like `font-weight` and `color` could be moved in to the generic headline styles (line 43)

Reply:
done